native/include/tcn.h

```diff
@@ -281,11 +281,20 @@ typedef struct {
 #define APR_INET6 APR_INET
 #endif
 
+#ifdef APR_UNIX
 #define GET_S_FAMILY(T, F)           \
     if (F == 0) T = APR_UNSPEC;      \
     else if (F == 1) T = APR_INET;   \
     else if (F == 2) T = APR_INET6;  \
+    else if (F == 3) T = APR_UNIX;   \
```

Comment:
Why not make the ifdef around this line?

Reply:
My compiler doesn't like this one - doesn't seem possible to put an ifdef inside a macro.

Reply:
Right, I completely missed this one that this is a macro. Apologies.